Support WG kernel module implementation if available (#9)
freifunkMUC/wg-embed: README.md

# wg-embed: an embeddable WireGuard-Go client

This module allows you to embed a WireGuard client into your Go application for easy tunnel setup.

> Uses the Go userspace implementationm of WireGuard by default, and optionally the Linux kernel module if it is found.

Reviewer: ```suggestion
Uses the Go userspace implementation of WireGuard by default, and optionally the Linux kernel module if available.
```

Author: Thanks, fixed
